docs/rules/group-exports.md

```diff
@@ -69,6 +69,12 @@ export const first = true
 export const second = true
 ```
 
+```js
+// Aggregating exports from the same module -> ok
```

Comment:
this should be "not ok", since you can do `export { module1, module2 } from 'module-1'`